Source/Core/Core/HW/GCKeyboard.cpp

```diff
@@ -18,7 +18,9 @@
 
 namespace Keyboard
 {
-static InputConfig s_config("GCKeyNew", _trans("Keyboard"), "GCKey");
+static constexpr int CONFIG_VERSION = 1;
```

Comment:
Unless I'm missing something, this value can just exist on the `InputConfig` class as a static and there isn't a need for the member variable.

Reply:
I guess it depends on if you want separate versions for each input config file.

Reply:
That's a good point, I didn't think of that.

I think it just depends on what the team wants to do long-term.  Moving to a static will collapse us into one config version.  This makes it easy to fold into a more general purpose config version later or still specialize it for inputs ala `INPUT_CONFIG_VERSION`.  It also would line up with other parts of the code like: `STATE_VERSION`, `CACHE_REVISION`, and `GX_PIPELINE_UID_VERSION`.  That makes knowing when to change the number easy.  Make a config change?  Bump the version.

However, separate versions has pros too.  Would love to hear others thoughts on this.

Reply:
If my thinking is correct, moving between a common version and separate versions for input config files has no consequences. Therefore, until there is a clear need for one or the other, which we choose is irrelevant.